src/components/Icon/Icon.stories.tsx

```diff
@@ -0,0 +1,130 @@
+import type { StoryObj } from "@storybook/react";
+import React from "react";
+// TODO: point to new color-tokens
+import * as ColorTokens from "../../../lib/tokens/ts/colors";
+import { ALL_ICONS } from "../../util/allIcons";
+import Text from "../Text";
+import { Icon, IconProps } from "./Icon";
+import styles from "./Icon.stories.module.css";
+
+export default {
+  title: "Icon",
+  component: Icon,
+  argTypes: {
+    name: {
+      control: {
+        type: "select",
+        options: ALL_ICONS,
+      },
+    },
+    color: {
+      control: {
+        type: "select",
+        options: ["currentColor", ...Object.keys(ColorTokens)],
+      },
+    },
+  },
+};
+
+export const Default: StoryObj<IconProps> = {
+  render: ({ name, color, ...rest }) => {
+    const computedColor = color && ColorTokens[color];
+    return <Icon {...rest} color={computedColor} name={name} />;
+  },
+  args: {
+    name: "close",
+    purpose: "decorative" as const,
+  },
+};
+
+export const Medium: StoryObj<IconProps> = {
+  ...Default,
+  args: {
+    ...Default.args,
+    size: "2em",
+  },
+};
+
+export const Large: StoryObj<IconProps> = {
+  ...Default,
+  args: {
+    ...Default.args,
+    size: "4em",
+  },
+};
+
+export const FullScreen: StoryObj<IconProps> = {
+  ...Default,
+  args: {
+    ...Default.args,
+    size: "100vh",
+  },
+  parameters: {
+    axe: {
+      disabledRules: ["scrollable-region-focusable"],
+    },
+  },
+};
+
+export const CustomColor: StoryObj<IconProps> = {
+  ...Default,
+  args: {
+    ...Default.args,
+    color: "EdsColorBrand400",
+    size: "2em",
+  },
+};
+
+export const InText: StoryObj<IconProps> = {
+  render: (args) => {
+    return (
+      <Text as="p">
```

Comment:
Just to check my understanding – `Text` renders a `p` tag be default, right?

Reply:
Yep, I'll remove redundancy, thanks for noticing